security/pkg/k8s/controller/customdnsname.go

```diff
@@ -20,6 +20,8 @@ import (
 	"istio.io/istio/pkg/log"
 )
 
+// ConstructCustomDNSNames creates DNS entries for given service accounts and allows
+// customization of the DNS names of specific service accounts
```

Comment:
would be helpful to indicate what and how customization will be generated

Reply:
agree. Added in additional commit.
@incfly PTAL